act-rules-format.bs

```diff
@@ -775,37 +773,28 @@ Examples in this section include:
   "hasPart": "dct:hasPart",
   "source": "dct:source",
 
-  "Agent": "foaf:term_Agent",
-  "Person": "foaf:term_Person",
-  "Organization": "foaf:term_Organization",
-  "Group": "foaf:term_Group",
-  "Document": "foaf:term_Document",
-  "name": "foaf:term_name",
-  "firstName": "foaf:term_firstName",
-  "surname": "foaf:term_surname",
-  "mbox": "foaf:term_mbox",
-  "mbox_sha1sum": "foaf:term_mbox_sha1sum",
-  "member": "foaf:term_member",
-  "homepage": "foaf:term_homepage",
-
-  "assertedBy": { "@id": "@type" },
```

Comment:
This might have been my mistake, but this needs to be  `{ "@type": "@id" }`. What we're saying here is that when assertedBy is a string instead of an object, the string will be an identifying URL for that object.